scripts/data/updateWidgetsOverview.ts

```diff
@@ -43,9 +43,12 @@ async function main(): Promise<void> {
                         }
                     );
 
-                    const hasPreview = await fs
+                    const hasStructurePreview = await fs
                         .readdir(join(packagePath, "src"))
                         .then(files => files.some(path => path.endsWith(".editorConfig.ts")));
+                    const hasDesignModePreview = await fs
+                        .readdir(join(packagePath, "src"))
+                        .then(files => files.some(path => path.endsWith(".editorPreview.tsx")));
```

Comment:
Having editorPreview file does not guarantee some component is being rendered. So we should check for the preview function.

Reply:
fair point. I now run the files through the typescript checker to see if the appropriate function is exported.